### ISSUE TITLE
NVDA does not read a full email.

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7196]
### Steps to reproduce:

I get email notifications from facebook and linkdyn, when opening these emails and start reading them with nvda + a, nvda reads the e-mail up to a certain point, but does not read the e-mail entirely.
To finish reading the email you must continue reading by pressing the arrows below.
It is expected that the email would be read entirely by pressing nvda + a.
It seems to me that the continuous reading is interrupted when there is a large stretch of empty lines.
Navigation mode is active.
Reading columns and table rows is disabled.
The log is below just like the version of nvda and other included software.

Input: kb(laptop):NVDA+a
IO - speech.speak (10:48:12):
Speaking [IndexCommand(1), IndexCommand(2), IndexCommand(3), u'\xa0 V\xe1rias coisas aconteceram no Facebook desde a \xfaltima vez que voc\xea entrou. ']
IO - speech.speak (10:48:12):
Speaking [IndexCommand(4), u'Essas s\xe3o algumas notifica\xe7\xf5es de seus amigos que voc\xea perdeu. \xa0 \xa0\xa0\xa0 ']
IO - speech.speak (10:48:12):
Speaking [IndexCommand(5), u"Fernando Silva \xa0 \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 10 mensagens \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 5 convites para grupos \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 7 atualiza\xe7\xf5es de grupo \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 7 op\xe7\xf5es ''Curtir'' em uma publica\xe7\xe3o em que voc\xea foi marcado \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 2 coment\xe1rios com foto \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 3 novos amigos \xa0 \xa0\xa0\xa0 \xa0\xa0\xa0 47 novas notifica\xe7\xf5es \xa0 \xa0\n", IndexCommand(6), IndexCommand(7), IndexCommand(8), IndexCommand(9), IndexCommand(10), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(11), IndexCommand(12), u'linque', u'Voc\xea tem novas notifica\xe7\xf5es.']
IO - speech.speak (10:48:25):
Speaking [u'fora de linque', u'\n', IndexCommand(13), IndexCommand(14), IndexCommand(15), IndexCommand(16), IndexCommand(17), IndexCommand(18), IndexCommand(19), IndexCommand(20), IndexCommand(21), IndexCommand(22), u'V\xe1rias coisas aconteceram no Facebook desde a \xfaltima vez que voc\xea entrou. ']
IO - speech.speak (10:48:25):
Speaking [IndexCommand(23), u'Essas s\xe3o algumas notifica\xe7\xf5es de seus amigos que voc\xea perdeu.\n']
IO - speech.speak (10:48:27):
Speaking [IndexCommand(24), IndexCommand(25), IndexCommand(26), IndexCommand(27), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(28), IndexCommand(29), u'linque', u'Fernando Silva', u'fora de linque', u'\n', IndexCommand(30), IndexCommand(31), IndexCommand(32), IndexCommand(33)]
IO - speech.speak (10:48:34):
Speaking [IndexCommand(34), IndexCommand(35), IndexCommand(36), IndexCommand(37), IndexCommand(38), IndexCommand(39), IndexCommand(40), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(41), IndexCommand(42), u'linque', u'10 mensagens', u'fora de linque', u'\n', IndexCommand(43), IndexCommand(44)]
IO - speech.speak (10:48:36):
Speaking [IndexCommand(45), IndexCommand(46), IndexCommand(47), IndexCommand(48), IndexCommand(49), IndexCommand(50), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(51), IndexCommand(52), u'linque', u'5 convites para grupos', u'fora de linque', u'\n', IndexCommand(53), IndexCommand(54), IndexCommand(55)]
IO - speech.speak (10:48:38):
Speaking [IndexCommand(56), IndexCommand(57), IndexCommand(58), IndexCommand(59), IndexCommand(60), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(61), IndexCommand(62), u'linque', u'7 atualiza\xe7\xf5es de grupo', u'fora de linque', u'\n', IndexCommand(63), IndexCommand(64), IndexCommand(65), IndexCommand(66)]
IO - speech.speak (10:48:41):
Speaking [IndexCommand(67), IndexCommand(68), IndexCommand(69), IndexCommand(70), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(71), IndexCommand(72), u'linque', u"7 op\xe7\xf5es ''Curtir'' em uma publica\xe7\xe3o em que voc\xea foi marcado", u'fora de linque', u'\n', IndexCommand(73), IndexCommand(74), IndexCommand(75), IndexCommand(76), IndexCommand(77)]
IO - speech.speak (10:48:44):
Speaking [IndexCommand(78), IndexCommand(79), IndexCommand(80), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(81), IndexCommand(82), u'linque', u'2 coment\xe1rios com foto', u'fora de linque', u'\n', IndexCommand(83), IndexCommand(84), IndexCommand(85), IndexCommand(86), IndexCommand(87), IndexCommand(88)]
IO - speech.speak (10:48:47):
Speaking [IndexCommand(89), IndexCommand(90), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(91), IndexCommand(92), u'linque', u'3 novos amigos', u'fora de linque', u'\n', IndexCommand(93), IndexCommand(94), IndexCommand(95), IndexCommand(96), IndexCommand(97), IndexCommand(98), IndexCommand(99)]
IO - speech.speak (10:48:49):
Speaking [IndexCommand(100), u'gr\xe1fico', u'linque', u' ', u'fora de linque', u'\n', IndexCommand(101), IndexCommand(102), u'linque', u'47 novas notifica\xe7\xf5es', u'fora de linque', u'\n', IndexCommand(103), IndexCommand(104), IndexCommand(105), IndexCommand(106), IndexCommand(107), IndexCommand(108), IndexCommand(109), IndexCommand(110)]
IO - inputCore.InputManager.executeGesture (10:49:05):
Input: kb(laptop):NVDA+f1
DEBUGWARNING - watchdog._watcher (10:49:06):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 199, in <module>
  File "core.pyc", line 391, in main
  File "wx\_core.pyc", line 8657, in MainLoop
  File "wx\_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 362, in Notify
  File "queueHandler.pyc", line 83, in pumpAll
  File "queueHandler.pyc", line 50, in flushQueue
  File "scriptHandler.pyc", line 145, in _queueScriptCallback
  File "scriptHandler.pyc", line 187, in executeScript
  File "globalCommands.pyc", line 1413, in script_navigatorObject_devInfo
  File "logging\__init__.pyc", line 1167, in info
  File "logHandler.pyc", line 122, in _log
  File "gui\logViewer.pyc", line 107, in activate
  File "wx\_core.pyc", line 9922, in Show

INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (10:49:06):
Developer info for navigator object:
name: u'Mensagem'
role: ROLE_DOCUMENT
states: STATE_MULTILINE, STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_OutlookWordDocumentIAccessibleWordDocumentWindowNVDAObject object at 0x0A54C6D0>
Python class mro: (<class 'NVDAObjects.Dynamic_OutlookWordDocumentIAccessibleWordDocumentWindowNVDAObject'>, <class 'appModules.outlook.OutlookWordDocument'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.winword.WordDocument'>, <class 'NVDAObjects.behaviors.EditableTextWithoutAutoSelectDetection'>, <class 'editableText.EditableTextWithoutAutoSelectDetection'>, <class 'NVDAObjects.behaviors.EditableText'>, <class 'editableText.EditableText'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: (8, 264, 1333, 455)
value: None
appModule: <'outlook' (appName u'outlook', process ID 9596) at address a552030>
appModule.productName: u'Microsoft Outlook'
appModule.productVersion: u'16.0.7369.2130'
TextInfo: <class 'NVDAObjects.window.winword.WordDocumentTextInfo'>
windowHandle: 67690
windowClassName: u'_WwG'
windowControlID: 0
windowStyle: 1442840576
windowThreadID: 8404
windowText: u'Mensagem'
displayText: u''
IAccessibleObject: <POINTER(IAccessible) ptr=0xeb87f38 at 56cf300>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=67690, objectID=-4, childID=0
IAccessible accName: u'Mensagem'
IAccessible accRole: ROLE_SYSTEM_CLIENT
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: None

### System configuration:
NVDA version:
next-14052,c80e529f

NVDA Installed or portable:
Installed 

Other information:

Windows version:
Windows 10 Version 1703 Build 15063.296
Name and version of other software in use when reproducing the issue:
Microsoft Outlook 2016 version 16.0.7369.2130

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
yes

**Comment by jcsteh** [comment]
What speech synthesiser are you using? Thanks.

**Comment by fernando-jose-silva** [comment]
Thank you very much, I am using the vocalizer esprecive Luciana's voice.

**Comment by PratikP1** [comment]
I have noticed a similar issue with Outlook 2016 emails not being read intermittently. I'm attaching two logs that show more details. Same outlook version with latest NVDA Next and windows 10 Pro 1703 build 16199. I'll file a separate report if these issues are not related. 
[nvda before restart - Outlook.txt](https://github.com/nvaccess/nvda/files/1051619/nvda.before.restart.-.Outlook.txt)
[nvda After restart - Outlook.txt](https://github.com/nvaccess/nvda/files/1051620/nvda.After.restart.-.Outlook.txt)

**Comment by PratikP1** [comment]
Possibly related to #6001 827?

**Comment by ehollig** [comment]
This should be fixed by #7777. Please comment if this is still an issue.
